source/index.d.ts

```diff
@@ -67,14 +69,11 @@ export type Modifiers =
 	| 'strikethrough'
 	| 'visible';
 
-/**
-Levels:
-- `0` - All colors disabled.
-- `1` - Basic 16 colors support.
-- `2` - ANSI 256 colors support.
-- `3` - Truecolor 16 million colors support.
-*/
-export type ColorSupportLevel = 0 | 1 | 2 | 3;
+export {
+	ColorInfo,
+	ColorSupport,
+	ColorSupportLevel,
+};
```

Comment:
```suggestion
} from 'supports-color';
```

I prefer explicit re-exporting.

And place it at the bottom of the file.